### ISSUE TITLE
keras import error when having a flatten layer

### ISSUE THREAD

**Issue by muyinanhai** [deeplearning4j/deeplearning4j#4695]
#### Issue Description
model.json:
{"class_name": "Model", "config": {"name": "model_1", "layers": [{"name": "cate_0", "class_name": "InputLayer", "config": {"batch_input_shape": [null, 1], "dtype": "int32", "sparse": false, "name": "cate_0"}, "inbound_nodes": []}, {"name": "cate_1", "class_name": "InputLayer", "config": {"batch_input_shape": [null, 1], "dtype": "int32", "sparse": false, "name": "cate_1"}, "inbound_nodes": []}, {"name": "cate_2", "class_name": "InputLayer", "config": {"batch_input_shape": [null, 1], "dtype": "int32", "sparse": false, "name": "cate_2"}, "inbound_nodes": []}, {"name": "cate_3", "class_name": "InputLayer", "config": {"batch_input_shape": [null, 1], "dtype": "int32", "sparse": false, "name": "cate_3"}, "inbound_nodes": []}, {"name": "cate_4", "class_name": "InputLayer", "config": {"batch_input_shape": [null, 1], "dtype": "int32", "sparse": false, "name": "cate_4"}, "inbound_nodes": []}, {"name": "cate_5", "class_name": "InputLayer", "config": {"batch_input_shape": [null, 1], "dtype": "int32", "sparse": false, "name": "cate_5"}, "inbound_nodes": []}, {"name": "cate_6", "class_name": "InputLayer", "config": {"batch_input_shape": [null, 1], "dtype": "int32", "sparse": false, "name": "cate_6"}, "inbound_nodes": []}, {"name": "cate_7", "class_name": "InputLayer", "config": {"batch_input_shape": [null, 1], "dtype": "int32", "sparse": false, "name": "cate_7"}, "inbound_nodes": []}, {"name": "cate_8", "class_name": "InputLayer", "config": {"batch_input_shape": [null, 1], "dtype": "int32", "sparse": false, "name": "cate_8"}, "inbound_nodes": []}, {"name": "cate_9", "class_name": "InputLayer", "config": {"batch_input_shape": [null, 1], "dtype": "int32", "sparse": false, "name": "cate_9"}, "inbound_nodes": []}, {"name": "cate_emb0", "class_name": "Embedding", "config": {"name": "cate_emb0", "trainable": true, "batch_input_shape": [null, null], "dtype": "float32", "input_dim": 2985, "output_dim": 10, "embeddings_initializer": {"class_name": "RandomUniform", "config": {"minval": -0.05, "maxval": 0.05, "seed": null}}, "embeddings_regularizer": null, "activity_regularizer": null, "embeddings_constraint": null, "mask_zero": false, "input_length": null}, "inbound_nodes": [[["cate_0", 0, 0, {}]]]}, {"name": "cate_emb1", "class_name": "Embedding", "config": {"name": "cate_emb1", "trainable": true, "batch_input_shape": [null, null], "dtype": "float32", "input_dim": 2852, "output_dim": 10, "embeddings_initializer": {"class_name": "RandomUniform", "config": {"minval": -0.05, "maxval": 0.05, "seed": null}}, "embeddings_regularizer": null, "activity_regularizer": null, "embeddings_constraint": null, "mask_zero": false, "input_length": null}, "inbound_nodes": [[["cate_1", 0, 0, {}]]]}, {"name": "cate_emb2", "class_name": "Embedding", "config": {"name": "cate_emb2", "trainable": true, "batch_input_shape": [null, null], "dtype": "float32", "input_dim": 2796, "output_dim": 10, "embeddings_initializer": {"class_name": "RandomUniform", "config": {"minval": -0.05, "maxval": 0.05, "seed": null}}, "embeddings_regularizer": null, "activity_regularizer": null, "embeddings_constraint": null, "mask_zero": false, "input_length": null}, "inbound_nodes": [[["cate_2", 0, 0, {}]]]}, {"name": "cate_emb3", "class_name": "Embedding", "config": {"name": "cate_emb3", "trainable": true, "batch_input_shape": [null, null], "dtype": "float32", "input_dim": 2778, "output_dim": 10, "embeddings_initializer": {"class_name": "RandomUniform", "config": {"minval": -0.05, "maxval": 0.05, "seed": null}}, "embeddings_regularizer": null, "activity_regularizer": null, "embeddings_constraint": null, "mask_zero": false, "input_length": null}, "inbound_nodes": [[["cate_3", 0, 0, {}]]]}, {"name": "cate_emb4", "class_name": "Embedding", "config": {"name": "cate_emb4", "trainable": true, "batch_input_shape": [null, null], "dtype": "float32", "input_dim": 2746, "output_dim": 10, "embeddings_initializer": {"class_name": "RandomUniform", "config": {"minval": -0.05, "maxval": 0.05, "seed": null}}, "embeddings_regularizer": null, "activity_regularizer": null, "embeddings_constraint": null, "mask_zero": false, "input_length": null}, "inbound_nodes": [[["cate_4", 0, 0, {}]]]}, {"name": "cate_emb5", "class_name": "Embedding", "config": {"name": "cate_emb5", "trainable": true, "batch_input_shape": [null, null], "dtype": "float32", "input_dim": 2722, "output_dim": 10, "embeddings_initializer": {"class_name": "RandomUniform", "config": {"minval": -0.05, "maxval": 0.05, "seed": null}}, "embeddings_regularizer": null, "activity_regularizer": null, "embeddings_constraint": null, "mask_zero": false, "input_length": null}, "inbound_nodes": [[["cate_5", 0, 0, {}]]]}, {"name": "cate_emb6", "class_name": "Embedding", "config": {"name": "cate_emb6", "trainable": true, "batch_input_shape": [null, null], "dtype": "float32", "input_dim": 2679, "output_dim": 10, "embeddings_initializer": {"class_name": "RandomUniform", "config": {"minval": -0.05, "maxval": 0.05, "seed": null}}, "embeddings_regularizer": null, "activity_regularizer": null, "embeddings_constraint": null, "mask_zero": false, "input_length": null}, "inbound_nodes": [[["cate_6", 0, 0, {}]]]}, {"name": "cate_emb7", "class_name": "Embedding", "config": {"name": "cate_emb7", "trainable": true, "batch_input_shape": [null, null], "dtype": "float32", "input_dim": 2662, "output_dim": 10, "embeddings_initializer": {"class_name": "RandomUniform", "config": {"minval": -0.05, "maxval": 0.05, "seed": null}}, "embeddings_regularizer": null, "activity_regularizer": null, "embeddings_constraint": null, "mask_zero": false, "input_length": null}, "inbound_nodes": [[["cate_7", 0, 0, {}]]]}, {"name": "cate_emb8", "class_name": "Embedding", "config": {"name": "cate_emb8", "trainable": true, "batch_input_shape": [null, null], "dtype": "float32", "input_dim": 2602, "output_dim": 10, "embeddings_initializer": {"class_name": "RandomUniform", "config": {"minval": -0.05, "maxval": 0.05, "seed": null}}, "embeddings_regularizer": null, "activity_regularizer": null, "embeddings_constraint": null, "mask_zero": false, "input_length": null}, "inbound_nodes": [[["cate_8", 0, 0, {}]]]}, {"name": "cate_emb9", "class_name": "Embedding", "config": {"name": "cate_emb9", "trainable": true, "batch_input_shape": [null, null], "dtype": "float32", "input_dim": 2597, "output_dim": 10, "embeddings_initializer": {"class_name": "RandomUniform", "config": {"minval": -0.05, "maxval": 0.05, "seed": null}}, "embeddings_regularizer": null, "activity_regularizer": null, "embeddings_constraint": null, "mask_zero": false, "input_length": null}, "inbound_nodes": [[["cate_9", 0, 0, {}]]]}, {"name": "continue_input", "class_name": "InputLayer", "config": {"batch_input_shape": [null, 10], "dtype": "float32", "sparse": false, "name": "continue_input"}, "inbound_nodes": []}, {"name": "flatten_1", "class_name": "Flatten", "config": {"name": "flatten_1", "trainable": true}, "inbound_nodes": [[["cate_emb0", 0, 0, {}]]]}, {"name": "flatten_2", "class_name": "Flatten", "config": {"name": "flatten_2", "trainable": true}, "inbound_nodes": [[["cate_emb1", 0, 0, {}]]]}, {"name": "flatten_3", "class_name": "Flatten", "config": {"name": "flatten_3", "trainable": true}, "inbound_nodes": [[["cate_emb2", 0, 0, {}]]]}, {"name": "flatten_4", "class_name": "Flatten", "config": {"name": "flatten_4", "trainable": true}, "inbound_nodes": [[["cate_emb3", 0, 0, {}]]]}, {"name": "flatten_5", "class_name": "Flatten", "config": {"name": "flatten_5", "trainable": true}, "inbound_nodes": [[["cate_emb4", 0, 0, {}]]]}, {"name": "flatten_6", "class_name": "Flatten", "config": {"name": "flatten_6", "trainable": true}, "inbound_nodes": [[["cate_emb5", 0, 0, {}]]]}, {"name": "flatten_7", "class_name": "Flatten", "config": {"name": "flatten_7", "trainable": true}, "inbound_nodes": [[["cate_emb6", 0, 0, {}]]]}, {"name": "flatten_8", "class_name": "Flatten", "config": {"name": "flatten_8", "trainable": true}, "inbound_nodes": [[["cate_emb7", 0, 0, {}]]]}, {"name": "flatten_9", "class_name": "Flatten", "config": {"name": "flatten_9", "trainable": true}, "inbound_nodes": [[["cate_emb8", 0, 0, {}]]]}, {"name": "flatten_10", "class_name": "Flatten", "config": {"name": "flatten_10", "trainable": true}, "inbound_nodes": [[["cate_emb9", 0, 0, {}]]]}, {"name": "merge_1", "class_name": "Merge", "config": {"name": "merge_1", "mode": "concat", "mode_type": "raw", "concat_axis": 1, "dot_axes": -1, "output_shape": null, "output_shape_type": "raw", "output_mask": null, "output_mask_type": "raw", "arguments": {}}, "inbound_nodes": [[["continue_input", 0, 0, {"mask": [null, null, null, null, null, null, null, null, null, null, null]}], ["flatten_1", 0, 0, {"mask": [null, null, null, null, null, null, null, null, null, null, null]}], ["flatten_2", 0, 0, {"mask": [null, null, null, null, null, null, null, null, null, null, null]}], ["flatten_3", 0, 0, {"mask": [null, null, null, null, null, null, null, null, null, null, null]}], ["flatten_4", 0, 0, {"mask": [null, null, null, null, null, null, null, null, null, null, null]}], ["flatten_5", 0, 0, {"mask": [null, null, null, null, null, null, null, null, null, null, null]}], ["flatten_6", 0, 0, {"mask": [null, null, null, null, null, null, null, null, null, null, null]}], ["flatten_7", 0, 0, {"mask": [null, null, null, null, null, null, null, null, null, null, null]}], ["flatten_8", 0, 0, {"mask": [null, null, null, null, null, null, null, null, null, null, null]}], ["flatten_9", 0, 0, {"mask": [null, null, null, null, null, null, null, null, null, null, null]}], ["flatten_10", 0, 0, {"mask": [null, null, null, null, null, null, null, null, null, null, null]}]]]}, {"name": "merge_2", "class_name": "Merge", "config": {"name": "merge_2", "mode": "concat", "mode_type": "raw", "concat_axis": 1, "dot_axes": -1, "output_shape": null, "output_shape_type": "raw", "output_mask": null, "output_mask_type": "raw", "arguments": {}}, "inbound_nodes": [[["continue_input", 0, 0, {"mask": [null, null]}], ["merge_1", 0, 0, {"mask": [null, null]}]]]}, {"name": "dense_1", "class_name": "Dense", "config": {"name": "dense_1", "trainable": true, "units": 256, "activation": "tanh", "use_bias": true, "kernel_initializer": {"class_name": "VarianceScaling", "config": {"scale": 1.0, "mode": "fan_avg", "distribution": "uniform", "seed": null}}, "bias_initializer": {"class_name": "Zeros", "config": {}}, "kernel_regularizer": null, "bias_regularizer": null, "activity_regularizer": null, "kernel_constraint": null, "bias_constraint": null}, "inbound_nodes": [[["merge_2", 0, 0, {}]]]}, {"name": "dense_2", "class_name": "Dense", "config": {"name": "dense_2", "trainable": true, "units": 128, "activation": "tanh", "use_bias": true, "kernel_initializer": {"class_name": "VarianceScaling", "config": {"scale": 1.0, "mode": "fan_avg", "distribution": "uniform", "seed": null}}, "bias_initializer": {"class_name": "Zeros", "config": {}}, "kernel_regularizer": null, "bias_regularizer": null, "activity_regularizer": null, "kernel_constraint": null, "bias_constraint": null}, "inbound_nodes": [[["dense_1", 0, 0, {}]]]}, {"name": "dense_3", "class_name": "Dense", "config": {"name": "dense_3", "trainable": true, "units": 11, "activation": "sigmoid", "use_bias": true, "kernel_initializer": {"class_name": "VarianceScaling", "config": {"scale": 1.0, "mode": "fan_avg", "distribution": "uniform", "seed": null}}, "bias_initializer": {"class_name": "Zeros", "config": {}}, "kernel_regularizer": null, "bias_regularizer": null, "activity_regularizer": null, "kernel_constraint": null, "bias_constraint": null}, "inbound_nodes": [[["dense_2", 0, 0, {}]]]}], "input_layers": [["continue_input", 0, 0], ["cate_0", 0, 0], ["cate_1", 0, 0], ["cate_2", 0, 0], ["cate_3", 0, 0], ["cate_4", 0, 0], ["cate_5", 0, 0], ["cate_6", 0, 0], ["cate_7", 0, 0], ["cate_8", 0, 0], ["cate_9", 0, 0]], "output_layers": [["dense_3", 0, 0]]}, "keras_version": "2.1.2", "backend": "theano"}

Exception in thread "main" org.deeplearning4j.nn.modelimport.keras.exceptions.UnsupportedKerasConfigurationException: Layer flatten_1 could not be mapped to Layer, Vertex, or InputPreProcessor. Please file an issue at http://github.com/deeplearning4j/deeplearning4j/issues.
	at org.deeplearning4j.nn.modelimport.keras.KerasModel.getComputationGraphConfiguration(KerasModel.java:335)
	at org.deeplearning4j.nn.modelimport.keras.KerasModel.getComputationGraph(KerasModel.java:374)
	at org.deeplearning4j.nn.modelimport.keras.KerasModel.getComputationGraph(KerasModel.java:363)
	at org.deeplearning4j.nn.modelimport.keras.KerasModelImport.importKerasModelAndWeights(KerasModelImport.java:198)
#### Version Information
* 0.9.2-SNAPSHOT
* mac os high sierra
